src/NServiceBus.AcceptanceTests/Routing/When_publishing_an_interface.cs

```diff
@@ -42,7 +43,8 @@ class Context : ScenarioContext
             public Type EventTypePassedToRouting { get; set; }
         }
 
-        class Publisher : EndpointConfigurationBuilder
+        [SuppressMessage("ReSharper", "MemberCanBePrivate.Global")]
```

Comment:
pretty sure we already have quite a few of those R# "warnings", so I'm not really whether we want to suppress this here?